clients/admin-ui/src/features/config-wizard/DescribeSystemsForm.tsx

```diff
@@ -82,9 +79,8 @@ const DescribeSystemsForm: NextPage<{
         });
       } else {
         toast.closeAll();
-        handleSystemFidesKey(values.fides_key);
-        handleChangeReviewStep(1);
-        handleChangeStep(4);
+        dispatch(setSystemFidesKey(values.fides_key ?? ""));
```

Comment:
Did this to satisfy type checking though I don't think it will actually come up on success. The `type FormValues = Partial<System>;` means it could be undefined, though elsewhere we were already using empty string for an unset key. 🤷